pkg/consolegraphql/filter/parser.y

```diff
@@ -193,6 +204,16 @@ value:
     $$ = NewNullVal()
   }
 
+value_list:
+ '(' value ')'
+  {
+    $$ = NewValueList($2)
+  }
+| '(' value_list ',' value ')'
```

Comment:
production rule `'(' value_list ',' value ')'` looks odd.   wouldn't that imply nesting of the brackets was required i.e.  (('A'), 'B') rather than ('A','B').